examples/bus_template.json

```diff
@@ -5,14 +5,12 @@
     "number_of_replicas" : 0
   },
   "mappings": {
-    "_default_": {
-      "_all": { "enabled": false },
-      "_source": { "compress": true },
-      "properties" : {
+    "_doc": {
```

Comment:
since _doc is only allowed in elasticsearch 6.2+ (prior to this types can't start with underscore), I'm a bit concerned with accepting this change, which won't work for anyone testing with a <6.2 es cluster. thoughts?

Reply:
__defaults__ is not allowed in elasticsearch 6.* 
So it's similar concern I suppose?

Reply:
unfortunately this is not obvious but `_default_` is treated differently. Installing a template with `_default_` on 6.1.2 will work, but `_doc` not.

That said, the template as it is doesn't work on 6.1.2 either because it still uses the string type which is no longer valid since 6.0